styles/style.css

```diff
@@ -1,21 +1,359 @@
-*{
-    margin:0;
-    padding:0;
-    
+* {
+    margin: 0;
+    padding: 0;
+    font-family: Arial, Helvetica, sans-serif;
+
 }
 
 header {
-    height: 150px;
+    background-image: url("../images/header.jpg");
+    height: 250px;
+
+}
+
+.header-1 {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-around;
+    align-content: center;
+    flex-wrap: wrap;
+}
+
+#logo {
+    margin: 20px;
+    padding-top: 30px;
+}
+
+h1 {
+    font-size: 40px;
+    padding-left: 20px;
+    padding-top: 20px;
+    padding-bottom: 20px;
+    color: #444444;
+    text-align: center;
+    background-color: #ebb722;
+
+}
+
+h2 {
+    font-size: 24px;
+    font-weight: bold;
+    padding-top: 30px;
+    padding-bottom: 30px;
+    text-align: center;
+
+}
+
+h3 {
+    text-align: center;
+    padding-bottom: 20px;
+}
+
+.form h2 {
+    text-align: left;
+}
+
+.form h3 {
+    padding-top: 10px;
+}
+
+.lessons h3,
+.private h3 {
+    margin-top: 20px;
+    padding: 15px;
+
+
+}
+
+.lessons h2,
+.private h2 {
+    margin-top: 20px;
+    padding: 15px;
+}
+
+.contact h3,
+.form h3 {
+    text-align: left;
+}
+
+
+nav ul {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-around;
+    align-content: center;
+    padding-top: 30px;
+    height: 50px;
+    line-height: 50px;
+
+
+
+}
+
+nav li {
+    list-style-type: none;
+    font-size: 25px;
+    margin-left: 20px;
+    margin: 20px;
+    padding-top: 30px;
+
+
+}
+
+nav a {
+    text-decoration: none;
+    color: white;
+}
+
+body {
+    font-size: 17px;
+}
+
+.about {
+    margin: 20px;
+}
+
+.home {
+    display: grid;
+    grid-template-columns: 7.5% 25% 5% 25% 5% 25% 7.5%;
+    margin-top: 50px;
+
+
 }
 
-.header {
+.home img {
     width: 100%;
+
+}
+
+.home h2,
+.lessons h3,
+.private h3 {
+    background-color: #1b447f;
+    color: white;
+}
+
+ul {
+    margin: 20px;
+}
+
+li {
+    list-style: inside;
+    padding-left: 20px;
+
+}
+
+.facilities {
+    grid-column-start: 2;
```

Comment:
By using 3 columns along with the column gap property mentioned above you can simplify these to be auto.